src/GafferSceneUIModule/LightEditorBinding.cpp

```diff
@@ -84,8 +84,8 @@ class LocationNameColumn : public StandardPathColumn
 
 		IE_CORE_DECLAREMEMBERPTR( LocationNameColumn )
 
-		LocationNameColumn()
-			:	StandardPathColumn( "Name", "name" )
+		LocationNameColumn( unsigned flags = PathColumn::ResizeStretch )
```

Comment:
This might just be because I've recently inherited a very wide monitor, but I'm not sure this is the ideal setting - it can introduce a lot of dead space between the light names and the parameters :

![image](https://user-images.githubusercontent.com/1133871/210400743-5bc589ee-4dc0-4499-a3e3-253e047b64fd.png)